### PR TITLE
Add Avalanche

### PR DESCRIPTION
**Pull request by akegaviar**
Added Avalanche docs

### CODE REVIEW

File: docs/platform/join-a-public-network.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ The node status will change from **Pending** to **Running** once deployed.
...
+## Join an Avalanche network
 
 1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
-1. Under **Blockchain protocol**, select **Tezos**.
-1. Under **Blockchain network**, select **Mainnet** or **Florencenet testnet**. Click **Next**.
+1. Under **Blockchain protocol**, select **Avalanche**.
+1. Under **Blockchain network**, select **Mainnet** or **Fuji testnet**. Click **Next**.
 1. Under **Type**, select whether to run a [shared](/glossary/shared-node) or a [dedicated](/glossary/dedicated-node) node.
-1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/tezos/modes).
+1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/bsc/modes).
```
Comment (easeev) [inline docs/platform/join-a-public-network.md:64]: BSC?

File: docs/operations/avalanche/modes.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+meta:
+  - name: description
+    content: Learn the difference between an Avalanche C-Chain full node and an archive node. Run sample commands to see the difference.
+  - name: keywords
+    content: avalanche avax node full archive difference
+---
+
+# Modes
+
+Chainstack supports deploying an Avalanche node on the C-Chain mainnet in the following modes:
```
Comment (easeev) [inline docs/operations/avalanche/modes.md:11]: I find it a little odd that we talk only about C-Chain here. Technically this doc is linked from the joining network wizard so customers could have the impression that we support only C-Chain

On a side note, do we update the copy in the wizard to explain this?

File: docs/platform/join-a-public-network.md (hunk 1 above)
Comment (akegaviar) [reply docs/platform/join-a-public-network.md:64]: My bad. Fixed.

File: docs/operations/avalanche/modes.md (hunk 2 above)
Comment (akegaviar) [reply docs/operations/avalanche/modes.md:11]: Makes sense. I redid the wording for a better explanation. Please check again.

Comment (akegaviar) [reply docs/operations/avalanche/modes.md]: Will submit copy wizard in the UI repo as discussed.

File: docs/operations/avalanche/modes.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+meta:
+  - name: description
+    content: Learn the difference between an Avalanche C-Chain full node and an archive node. Run sample commands to see the difference.
+  - name: keywords
+    content: avalanche avax node full archive difference
+---
+
+# Modes
+
+Chainstack supports deploying an Avalanche node on the mainnet in the following modes:
```
Comment (easeev) [inline docs/operations/avalanche/modes.md:11]: Still not clear by reading this whether I'll get access to X-Chain or not. We're talking only about C-Chain here

Comment (akegaviar) [reply docs/operations/avalanche/modes.md:11]: I think the docs are accurate in reflecting what's currently there. To my knowledge, the definition of "archive" mode on Avalanche applies only to C-Chain.

So when we deploy an Avalanche node in archive mode we are deploying the EVM-based C-Chain with the archive of the historical states. That's pretty much it. To my knowledge.

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**